src/app/+directory/directory.component.html

```diff
@@ -46,7 +46,7 @@
         </a>
       </ng-template>
     </div>
-    <scholars-pagination [page]="page" [ngSwitch]="directoryView.layout" [pageSizeOptionsType]="'list'">
+    <scholars-pagination [page]="page" [ngSwitch]="directoryView.layout" [pageSizeOptionsType]="'list'" >
```

Comment:
```suggestion
    <scholars-pagination [page]="page" [ngSwitch]="directoryView.layout" [pageSizeOptionsType]="'list'">
```

Remove trailing space.